lib/store_web/templates/starbook/index.html.eex

```diff
@@ -0,0 +1,71 @@
+<div class="starbook">
+  <h1>StarBook <small>Mk V</small></h1>
+  <p>with elementary OS</p>
```

Comment:
Can we reiterate the starting price here?

Reply:
Problem is I don't think we can reliably keep it in sync; I _think_ it changes with currency conversion and whatnot.